changelog/deprecate_dualcontext.dd

```diff
@@ -1,13 +1,9 @@
 Deprecate local templates that receive local symbols by alias
 
-Support for local and member templates which can be instantiated with local
-symbols was added to the language in D 2.087.0, however there are a number of
-issues with the feature including:
+Support for instantiating local and member templates with local
+symbols was implemented in DMD 2.087.0. However, the implementation
+was incompatible with GDC and LDC backends.
 
-- Only DMD has code-gen support for the feature.
-- There has been no support to implement this feature in either GDC or LDC.
-- There is still no language specification that describes the feature.
-
-Because of this, there has been a dead feature introduced to the language for
-over nine releases, so now it has been deprecated, and due to be fully reverted
-in a future release.
+In order to maintain feature parity among D implementations, this
+improvement has been deprecated, and may be removed from a future DMD
```

Comment:
I understand that the removal is conditional on that the compiler vendors fail to implement the feature in their backends.

Reply:
This isn't being hostile at all. You are reading between lines here.

Reply:
The usage of phrases like "dead feature" and "over nine releases" have a negative tone and the rephrasing is much more appropriate to the professional, neutral tone of an official document.

Reply:
>
>The usage of phrases like "dead feature" and "over nine releases" have a negative tone
The negative note that is the result of your interpretation here. Not everyone have the same interpretation here.

Reply:
Considering the context being a professional software project changelog, it does clearly read aggravated/frustrated to me. We do not want to convey that sentiment to users.